include/target-sys/bf_sal/bf_sys_log.h

```diff
@@ -78,11 +78,9 @@ extern "C" {
 #define BF_MOD_BFRT (BF_MOD_START + 19)
 #define BF_MOD_P4RT (BF_MOD_START + 20)
 #define BF_MOD_SWITCHD (BF_MOD_START + 21)
-#define KRNLMON (BF_MOD_START + 22)
-#define OVSP4RT (BF_MOD_START + 23
-#define INFRAP4D (BF_MOD_START + 24)
```

Comment:
Those modules are added by Nupur. Just forward to Nupur to review it for OVS side.

Reply:
I believe Nupur is still on marital leave. I'm TL on the project.

Reply:
Come to think of it, I could mitigate the KRNLMON change by defining `KRNLMON` as a synonym for the new symbol and marking the old name as deprecated.

Reply:
... which I have now done.